config/optional/block.block.localgov_guides_contents.yml

```diff
@@ -0,0 +1,20 @@
+uuid: e5c7f023-2929-42b7-861a-3796d0808ec4
```

Comment:
This one wants deleting.

Reply:
Yup - missed that one. It's now been removed.